specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/gallery.json

```diff
@@ -1069,6 +1069,20 @@
       ],
       "description": "Describes the basic gallery artifact publishing profile."
     },
+    "TargetRegion": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the region."
+        },
+        "regionalReplicaCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "This is the number of source blob copies in this specific region."
```

Comment:
```source blob``` in the source region, or in the target region? Or ```target blob```? Please consider revising the statement to make it clearer.

Reply:
So it is the copies of the source blob in the target regions. Will this be better "This is the number of copies of the source blobs in this target region."?

Reply:
OK

Reply:
@vanbasten2323  what exactly is the "number of copies of the source blobs mean"?

Reply:
@singhkays, the original mail communicates the new name would be `scaleTier`, so i am bit confused now. Can you guys offer a bit of explanation, so i can communicate better to CLI users

Reply:
Also, what is the service end default value for this?

Reply:
Clarified in offline call